cves/kernel/CVE-2021-38204.yml

```diff
@@ -55,7 +55,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Users that are physically using a linux machine with this software could cause denial of service if they remove
```

Comment:
Which software? On the website, viewers will have no context at all to what you're talking about.

Reply:
updated description to be more clear